repeat-loop.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+while true
+do
+  exec kube-bench
```

Comment:
Need to pass the parameters through (for example if someone specifies `--version`)

Reply:
To try this out I had to add the repeat-loop.sh to the Dockerfile